--- SigUtil.cpp	2017-12-19 12:38:53.822035003 +0100
+++ SigUtil-new.cpp	2017-12-19 13:05:07.722028999 +0100
@@ -13,7 +13,7 @@
 
 #include <execinfo.h>
 #include <csignal>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/prctl.h>
 #include <sys/stat.h>
 #include <sys/uio.h>
@@ -189,19 +189,21 @@
         action.sa_handler = SIG_DFL;
 
         sigaction(signal, &action, nullptr);
-
+#ifdef HAVE_BACKTRACE_SYMBOLS 
         dumpBacktrace();
-
+#endif /* HAVE_BACKTRACE_SYMBOLS */
         // let default handler process the signal
         kill(Poco::Process::id(), signal);
     }
 
+#ifdef HAVE_BACKTRACE_SYMBOLS 
     void dumpBacktrace()
     {
         char header[32];
         sprintf(header, "Backtrace %d:\n", getpid());
 
         const int maxSlots = 50;
+
         void *backtraceBuffer[maxSlots];
         int numSlots = backtrace(backtraceBuffer, maxSlots);
         if (numSlots > 0)
@@ -226,6 +228,7 @@
                 }
             }
         }
+       
 
         if (std::getenv("LOOL_DEBUG"))
         {
@@ -233,6 +236,7 @@
             sleep(30);
         }
     }
+ #endif /* HAVE_BACKTRACE_SYMBOLS */
 
     void setFatalSignals()
     {
